### DISCUSSION TITLE
W11 24h2 update to Atlas?

### DISCUSSION THREAD

**Discussion by fireant21** [Atlas-OS/Atlas#1238] [Help & Support] [answered]
It seems Windows 11 version 24h2 is out now. I've been using Atlas on 23h2 for a year now, and would like to do a clean install of my OS. Is Atlas updating to match this new version of W11, and should I wait for it to reinstall? Thanks kindly.

**Comment by Ast3risk-ops** [comment]
0.5 will support 24h2, wait for it to release.

**Reply by AudiEtoffe** [reply]
Do you think .5 on 24h2 will be upgradable from .4 on a previous windows service pack?

**Reply by Ast3risk-ops** [reply]
I don't know, you'll have to see when it's announced.

**Reply by fireant21** [reply]
Alright thanks for the info!

**Comment by shader69** [comment]
Do you know when version 0.5.0 will be released? I got a brand-new PC today and updated to 24H2 before I could install Atlas...

**Reply by pencilnav** [reply]
We will be supporting it soon, but right now there's no ETA.

**Reply by rapidfire-zyrex** [reply]
HOW MUCH IT WOULD TAKE TO SUPPORT.

**Reply by WarmXR** [reply]
> HOW MUCH IT WOULD TAKE TO SUPPORT.

Chill out

**Comment by he3als** [comment, marked as answer]
Completed in v0.4.1
